test/reading-time.js

```diff
@@ -155,4 +162,33 @@ describe('readingTime()', function() {
     minutes: 2,
     time: 120000
   }))
+
+  it('should handle a CJK paragraph',
+  test('你好！这是一段中文。', {}, {
+    text: '1 min read',
+    words: 10,
```

Comment:
The word count should be 8 instead of 10 since we don't treat full-width punctuations as a CJK word.